src/paperless/settings.py

```diff
@@ -420,19 +420,30 @@ def _parse_beat_schedule() -> dict:
     # regular login in case the provided user does not exist.
     MIDDLEWARE.insert(_index + 1, "paperless.auth.AutoLoginMiddleware")
 
-ENABLE_HTTP_REMOTE_USER = __get_boolean("PAPERLESS_ENABLE_HTTP_REMOTE_USER")
-HTTP_REMOTE_USER_HEADER_NAME = os.getenv(
-    "PAPERLESS_HTTP_REMOTE_USER_HEADER_NAME",
-    "HTTP_REMOTE_USER",
-)
 
-if ENABLE_HTTP_REMOTE_USER:
-    MIDDLEWARE.append("paperless.auth.HttpRemoteUserMiddleware")
-    AUTHENTICATION_BACKENDS.insert(0, "django.contrib.auth.backends.RemoteUserBackend")
-    REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"].append(
-        "rest_framework.authentication.RemoteUserAuthentication",
+def _parse_remote_user_settings() -> str:
```

Comment:
What this to allow some testing?

Reply:
Yep, exactly

Reply:
Ok, always good to test

- Looks like ENABLE_HTTP_REMOTE_USER isn't used outside the function, so it can be lower case for clarity of not being a real setting
- I think to "properly" change the globals, they should be marked with the `global` keyword.  So under the if, a `global MIDDLEWARE`, etc

Reply:
Ah, ok thanks